x-pack/plugins/observability/public/pages/rule_details/utils.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { formatDurationFromTimeUnitChar, TimeUnitChar } from '../../../common';
+
+export const formatInterval = (ruleInterval: string) => {
```

Comment:
it will be nice to have some simple unit test around this function

Reply:
Noted. Will add tests in a separate PR